### PR TITLE
FIX: Substituted legacy DataObject::db() for DataObject::getSchema()

### PR DESCRIPTION
**Pull request by phptek**

### CODE REVIEW

File: src/Actions/NotifyUsersWorkflowAction.php [hunk 1]
```diff
@@ -164,7 +164,7 @@ public function getContextFields(DataObject $target)
             return $result;
         }
 
-        $fields = $target->db();
+        $fields = $target->getSchema()->fieldSpecs($target);
```
Comment (robbieaverill) [inline src/Actions/NotifyUsersWorkflowAction.php]: Any reason for choosing this method over say `DataObjectSchema::databaseFields()`?

Comment (phptek) [reply src/Actions/NotifyUsersWorkflowAction.php]: 1). Not really but (see 2)
2). Seeing as $target already has a schema accessible, seems wasteful to re-instantiate it statically
3). I was looking for something like:

```
$fields = $target->databaseFields(); // <-- No need to pass anything as an arg here
```

Comment (phptek) [reply src/Actions/NotifyUsersWorkflowAction.php]: Aka `$target->db()` in SS3 :-P

Comment (robbieaverill) [reply src/Actions/NotifyUsersWorkflowAction.php]: Sorry, the changelog is misleading. This is the suggested drop in replacement =)

**Pull request merged**